flycheck.el

```diff
@@ -6390,6 +6391,14 @@ See URL `http://www.jscs.info'."
                    (flycheck-sanitize-errors errors)))
   :modes (js-mode js2-mode js3-mode))
 
+(flycheck-define-checker javascript-standard
+  "A code style linter for the JavaScript Standard Style.
+
+See URL `https://github.com/feross/standard'."
+  :command ("standard" source)
```

Comment:
I read that `standard` is based on eslint, so I was wondering whether it probably has an option for checkstyle output, too?